### ISSUE TITLE
Set schema owner for version table schema

### ISSUE THREAD

**Issue by josh-hoiland** [fluentmigrator/fluentmigrator#1517]
**Is your feature request related to a problem? Please describe.**
I have a custom implementation of IVersionTableMetaData where I am setting SchemaName. And then, in my migrations, I'm creating tables in the same schema, and I'd like to be able to set the schema owner. It looks like if I create the schema from within a migration, I can set the owner with the ICreateSchemaOptionSyntax.Authorization extension method, but since the schema is being created by the version table migrations, this fails.

**Describe the solution you'd like**
Add another property to IVersionTableMetaData for schema owner

**Is this a feature you'd like to submit a PR for?**
no

**Comment by jzabroski** [comment]
Can you alter the authorization after it has been created instead?

**Comment by josh-hoiland** [comment]
Yeah, I should have mentioned that's what I'm doing as a workaround.
